api/src/main/resources/liquibase-update-to-2.0.xml

```diff
@@ -9485,6 +9491,8 @@
     </changeSet>
 
     <changeSet id="TRUNK-3422-20160216-1704" author="PralayRamteke" >
+		<validCheckSum>3:b4caee668484f1d2759bbf3feeba2daa</validCheckSum> <!-- old checksum when "Get" -->
```

Comment:
What do you mean by "old checksum when Get"?

Reply:
Sorry just typo